include/swift/Frontend/Frontend.h

```diff
@@ -433,6 +449,57 @@ class CompilerInstance {
   /// Frees up the ASTContext and SILModule objects that this instance is
   /// holding on.
   void freeContextAndSIL();
+
+private:
+  /// Load stdlib & true if should continue, i.e. no error
+  bool loadStdlib();
+  ModuleDecl *importUnderlyingModule();
+  ModuleDecl *importBridgingHeader();
+
+  void
+  getImplicitlyImportedModules(SmallVectorImpl<ModuleDecl *> &importModules);
+
+public: // for static functions in Frontend.cpp
+  struct ImplicitImports {
+    SourceFile::ImplicitModuleImportKind kind;
+    ModuleDecl *objCModuleUnderlyingMixedFramework;
+    ModuleDecl *headerModule;
+    SmallVector<ModuleDecl *, 4> modules;
+
+    explicit ImplicitImports(CompilerInstance &compiler);
+  };
+
+private:
+  void createREPLFile(ImplicitImports &implicitImports);
```

Comment:
Could be `const`, right?

(Const-correctness isn't part of our formal coding guidelines, but it's a habit I like to get into, at least with types that represent values, rather than processes or "controllers" like CompilerInstance.)

Reply:
Yup! I had some earlier code that used a const on an OptionSet that was passed in, and IIRC you nixed that.

Reply:
But I agree, const is goodness.

Reply:
The difference in this case is the reference. We usually omit `const` for value types passed by value, even though an assignment within the function could cause trouble. But if a parameter's passed by reference the `const`-ness is relevant to callers as well.

Reply:
Got it. Yes, I was nit-pickingly pointing out the "even though" part, because I have seen some long functions, where a potential modification in an argument up top could change what is seen near the end and it would be hard to see. Not an issue for short functions.